extmod/asyncio/stream.py

```diff
@@ -146,14 +175,24 @@ async def _serve(self, s, cb):
             except:
                 # Ignore a failed accept
                 continue
+            if s2:
+                if ssl and _ssl:
+                    if isinstance(ssl, _ssl.SSLContext):
+                        s2.setblocking(True)
+                        try:
+                            s2 = ssl.wrap_socket(s2, server_side=True)
+                        except OSError as e:
+                            sys.print_exception(e)
+                            s2.close()
```

Comment:
Does this behaviour match CPython, of printing the exception to stdout and then continuing?

Reply:
Not sure about how CPython manages this but after some testing this was the less resource intensive method I could think of, e.g.  the other alternative 
```python
while True:
    try:
          await asyncio.start_server(...)
     except Exception:
            ....
```
creates a `Server` instance and a task every time a handshake fails due to the client using HTTP  or not trusting the server certificate....